### PR TITLE
Basic Pool deployment 🏊🏊

### PR DESCRIPTION
**Pull request by TheRealSeber**

### CODE REVIEW

File: app/api/routers/pool.py [hunk 1]
```diff
@@ -150,3 +150,23 @@ async def deploy_pool(pool_id: str, db: Session = Depends(get_db)):
 
     await PoolService.deploy_pool(db=db, pool_id=pool_id)
     return JSONResponse({"detail": "Pool deployment started."})
+
+
+@router.post("/{pool_id}/deactivate", status_code=200)
+async def deactivate_pool(pool_id: str, db: Session = Depends(get_db)):
+    """
+    Deploy the pool with the given ID.
```
Comment (HermanPlay) [inline app/api/routers/pool.py:158]: Documentation differs from the actual functionality

File: app/api/services/pool.py [hunk 2]
```diff
@@ -216,3 +219,17 @@ async def deploy_pool(cls, db: MongoClient, pool_id: str) -> None:
 
         pool_deployment = PoolDeployment(name=pool.name, models=models)
         pool_deployment.deploy()
+
+    @classmethod
+    async def deactivate_pool(cls, db: MongoClient, pool_id: str) -> None:
+        """
+        Deploys a pool.
```
Comment (HermanPlay) [inline app/api/services/pool.py:226]: Documentation different from the actual functionality

**Review by HermanPlay — CHANGES_REQUESTED** [review]
Small changes needed

File: app/api/charts/pool/templates/virtualservice.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  hosts:
+  - "*"
+  gateways:
+  - {{ .Values.gateway }}
+  http:
+  - match:
+    - uri:
+        prefix: "/{{ .Values.prefix }}/"
+    - uri:
+        prefix: "/{{ .Values.prefix }}"
```
Comment (TheRealSeber) [inline app/api/charts/pool/templates/virtualservice.yaml:16]: why did you write the same thing twice?

Comment (matined) [reply app/api/charts/pool/templates/virtualservice.yaml:16]: it's not the same thing

**Pull request merged**